### PR TITLE
Refactor AuthUserService

### PR DESCRIPTION
**Pull request by awinabi**
@grassiricardo Refactored some code in Auth service. Can you please review and merge if found appropriate ?

### CODE REVIEW

File: app/services/v1/auth_user_service.rb [hunk 1]
```diff
@@ -5,36 +5,36 @@ module V1
...
+        return ERROR_URL
       end
     end
 
     def authentication
-      @token = Base58.encode(SecureRandom.uuid.delete('-').to_i(16))
-      response = new_instance_firebase.push("users_authenticated", {
+      token = Base58.encode(SecureRandom.uuid.delete('-').to_i(16))
+      response = firebase_client.push('users_authenticated', {
         :email => @email,
```
Comment (henriquepjv) [inline app/services/v1/auth_user_service.rb]: Já que vc possui o `attr_reader :email` não seria mais seguro trocar aonde aparece `@email` por apenas `email` ?

Comment (awinabi) [reply app/services/v1/auth_user_service.rb]: Totally agree!

**Comment by grassiricardo** [comment]
@diogopms Could you review this?

**Comment by diogopms** [comment]
LGTM 👍

**Comment by grassiricardo** [comment]
@awinab, @diogopms and @henriquepjv  Tranks!

**Pull request merged**